src/commonMain/kotlin/fr/acinq/lightning/db/PaymentsDb.kt

```diff
@@ -387,6 +388,21 @@ data class SpliceCpfpOutgoingPayment(
     override val completedAt: Long? = confirmedAt
 }
 
+data class InboundLiquidityOutgoingPayment(
+    override val id: UUID,
+    override val channelId: ByteVector32,
```

Comment:
The mining fees should be provided explicitly and set the same way as for `SpliceOutgoingPayment`, currently we only count the mining fees for remote funding.

```suggestion
    override val miningFees: Satoshi,
    override val channelId: ByteVector32,
```

Reply:
Proposed fix in #569.

Reply:
Good catch!

There is something I'm not 100% sure about. For every other splice transaction, the wallet user pays for *all* the mining fees. But for liquidity ads, this may be different: what the user pays in mining fees may be less than the actual transaction's mining fees, or more, depending on how many inputs/outputs are added by the LSP and what was set in the lease's `funding_weight`. Should the `miningFees` field match the transaction's mining fees, or should it match what was paid by the user? I chose the second option in https://github.com/ACINQ/lightning-kmp/pull/561/commits/df15490d105fc02d4e2fc9fa3ace49cfa409469e, while your proposal chose the first one.

Reply:
> I agree with option 2, however I would have defined miningFees as a normal attribute and set them to miningFees = action.fundingTx.sharedTx.tx.localFees.truncateToSatoshi().

If we do that, we should set `miningFees = action.fundingTx.sharedTx.tx.localFees.truncateToSatoshi() + lease.miningFee`, right? We want that field to include all fees that are for on-chain stuff, don't we? The goal is to cleanly separate them from the service fees?

Reply:
I think https://github.com/ACINQ/lightning-kmp/pull/561/commits/b77df8cfcc9a7446eb0c203ceffef622c9648c1f should remove the confusion, and fix other types of splices (even though in their case, `localFees` currently equals `fees`, but it's more future-proof to only use `localFees` in case we later include unrelated inputs/outputs on the LSP side).